draft-ietf-rtcweb-jsep.xml

```diff
@@ -3534,7 +3538,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             m= section MUST contain an "a=rtcp-mux" attribute. If an
             "m=" section contains an "a=rtcp-mux-only" attribute then
             that section MUST also contain an "a=rtcp-mux"
-            attribute.</t>
+            attribute. If this m= section was present in the previous
```

Comment:
This is somewhat confusing because this paragraph starts with talking about mandatory mux. It might work better as its own para.